build_tools/third_party/cuda/CMakeLists.txt

```diff
@@ -6,16 +6,36 @@
 
 set(CUDA_HEADERS_API_ROOT "${IREE_ROOT_DIR}/third_party/cuda/include")
 
-external_cc_library(
-  PACKAGE
-    cuda_headers
-  NAME
-    cuda_headers
-  ROOT
-    ${CUDA_HEADERS_API_ROOT}
-  HDRS
-    "cuda.h"
-  INCLUDES
-    ${CUDA_HEADERS_API_ROOT}
-)
+message("IREE_ENABLE_CUPTI: ${IREE_ENABLE_CUPTI}")
+if (TARGET CUDA::cupti)
+ message("cuda target: CUDA::cupti")
```

Comment:
remove debugging messages and ensure that bazel_to_cmake is still running on this file